include/swift/SIL/SILModule.h

```diff
@@ -132,6 +132,10 @@ class SILModule {
   /// The list of SILVTables in the module.
   VTableListType vtables;
 
+  /// This is a cache of vtable entries for quick look-up
+  llvm::DenseMap<const SILVTable *, llvm::DenseMap<SILDeclRef, SILFunction *>>
+      VTableEntryCache;
+
```

Comment:
Anyway, instead of this outer DenseMap you can directly place the inner DenseMap into SILVTable directly.